### PR TITLE
EES-4738 - Document all the environment variables used in Kangal

### PR DESCRIPTION
**Pull request by s-radyuk**
This PR adds a new document which contains all the env vars used to configure Kangal.
Documentation for helm chart was also updated.
Some unused and duplicated variables were removed from configs.
Also superlinter version was updated in the testing workflow.

### CODE REVIEW

File: charts/kangal/README.md [hunk 1]
```diff
@@ -9,20 +9,20 @@ This chart bootstraps a [kangal](https://github.com/hellofresh/kangal) deploymen
...
 ```shell
-$ helm install kangal kangal/kangal
+ helm install kangal kangal/kangal
 ```
 
 > **Tip**: You can provide values on the command line with `--set` or using the `values` file with `-f my-values-file.yaml`. Eg.:
 To set AWS credentials using command line you can use the following flags:
 
-```
+```yaml
```
Comment (vgarvardt) [inline charts/kangal/README.md:25]: this is not yaml, I think, looks like shell command flags

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -24,14 +24,15 @@ jobs:
           version: v1.42.1
 
       - name: Run General Linting
-        uses: docker://ghcr.io/github/super-linter:slim-v4
+        uses: docker://ghcr.io/github/super-linter:slim-v4.8.5
```
Comment (vgarvardt) [inline .github/workflows/testing.yml:27]: in theory this should be the same images - https://github.com/github/super-linter/pkgs/container/super-linter
image

But I'm fine with pinning the full version as it can be some sort of caching preventing it from using really this latest version

File: .github/workflows/testing.yml [hunk 3]
```diff
@@ -24,14 +24,15 @@ jobs:
...
         env:
           VALIDATE_ALL_CODEBASE: false
           VALIDATE_GO: false
           VALIDATE_JSCPD: false
           VALIDATE_GITHUB_ACTIONS: false
           VALIDATE_NATURAL_LANGUAGE: false
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VALIDATE_KUBERNETES_KUBEVAL: false
```
Comment (vgarvardt) [inline .github/workflows/testing.yml:35]: Probably we should switch from opt-out model to opt-in and list all the things we want to lint explicitly. Btw - why do we have `VALIDATE_GO: false` if this is the main project language? =) Anyway, should be done in own PR

**Review by vgarvardt — APPROVED** [review]

File: .github/workflows/testing.yml (hunk 3 above)
Comment (s-radyuk) [reply .github/workflows/testing.yml:35]: `VALIDATE_GO: false` is because we use golangci-lint-action for linting go. And after that we use superlinter for all the rest.

Comment (vgarvardt) [reply .github/workflows/testing.yml:35]: `super-linter` uses `golangci`under the hood, so it makes sense to use the super one
